packages/flutter/test/widgets/editable_text_cursor_test.dart

```diff
@@ -1236,7 +1236,7 @@ void main() {
       backgroundCursorColor: Colors.grey,
       controller: controller,
       focusNode: focusNode,
-      style: const TextStyle(),
+      style: const TextStyle(fontSize: 17),
```

Comment:
Why set this fontSize now?

Reply:
Because otherwise this test is failing.
This test sets cursorHeight to 17 and did not set the font size (it defaults to 14). This is the case covered by the second test I added (cursorHeight bigger than font size).
In this case, because the cursor is bigger than the font size it will overflow the editable vertically. Before my PR the overflow was only on the bottom (the cursor was painted with a vertical position align to the top of the text and overflow on the bottom).
With my PR, when a cursorHeight is bigger than the font size, the cursor is centered so it overflows on the top and on the bottom).
I choose to change the fontSize in this test instead of changing the rect that it tested some lines later because I don't think that checking that the cursor did not overflow vertically was the goal of this test (and because I added a test for this particular case) .

Reply:
Thanks for the thorough explanation, I agree that this looks to be in line with the original intentions of this test :+1: